### PR TITLE
Fix weather problem.

### PR DESCRIPTION
**Pull request by duncangrubbs**

### CODE REVIEW

File: src/scripts/EventPage.js [hunk 1]
```diff
@@ -38,10 +38,10 @@ class EventPage {
       .then(position => {
         const WEATHER_RESOURCE =
           'http://api.openweathermap.org/data/2.5/weather';
-        const API_KEY = '55c2586d12873c5d39e99b0dea411dc2';
+        const API_KEY = '822795ff3e00fd43e5d4596cc0849649';
         let lat = position.coords.latitude;
         let long = position.coords.longitude;
-        let qry = `lat=${lat}&lon=${long}&APPID=${API_KEY}&units=metric`;
+        let qry = `&lat=${lat}&lon=${long}&APPID=${API_KEY}&units=metric`;
```
Comment (AlexanderOtavka) [inline src/scripts/EventPage.js:44]: Why did you add the `&` here?  It should work without.

Comment (duncangrubbs) [reply src/scripts/EventPage.js:44]: That is what fixed it I am pretty sure

**Comment by AlexanderOtavka** [comment]
Fixed on master
